src/gui/ogr/qgsnewogrconnection.h

```diff
@@ -45,7 +44,7 @@ class GUI_EXPORT QgsNewOgrConnection : public QDialog, private Ui::QgsNewOgrConn
   public slots:
     void accept() override;
     void on_btnConnect_clicked();
-    void on_buttonBox_helpRequested() { QgsContextHelp::run( metaObject()->className() ); }
+    void helpRequest();
```

Comment:
What's your opinion on renaming this to `showHelp ()`?

Not sure if this actually should be public API. It could as well be private for me. If it stays public, it needs proper doxygen comments.